app/models/location.rb

```diff
@@ -1,4 +1,10 @@
+require 'securerandom'
```

Comment:
I use this to generate a random password for when a vendor is created and associated with a farmer's market location